tic_tac_toe.rb

```diff
@@ -1,13 +1,15 @@
 require './lib/game_session'
 require './lib/player'
 require './lib/board'
-require './lib/rules'
+require './lib/outcome'
 require './lib/display'
+require './lib/initializer'
 
 player1 = Player.new(' X')
 player2 = Player.new(' O')
 display1 = Display.new
```

Comment:
Out of curiousity, why did you name this `display1`?

Reply:
I named it display1 because sublime highlighted 'display' as a keyword. I thought maybe the word was a special keyword that would act differently. I changed display1 to display and nothing broke though so now I know :)